src/main/java/org/truffleruby/core/VMPrimitiveNodes.java

```diff
@@ -174,7 +174,7 @@ public VMMethodLookupNode() {
         @Specialization
         public DynamicObject vmMethodLookup(VirtualFrame frame, Object self, Object name) {
             // TODO BJF Sep 14, 2016 Handle private
```

Comment:
Is this TODO still valid? Your commit message mentions calling `to_str` privately.

Reply:
@bjfish Was this TODO about calling `to_str` privately? Or something else?

Reply:
I think this TODO is about added a boolean private parameter which would allow including/excluding private methods from look. I'm guessing this would be to recreate some of the functionality around here:
https://github.com/ruby/ruby/blob/e07613e27ee9445abad9c6981e8d721a0df39f1c/vm_method.c#L1971 and other related calls.

Reply:
Mmh, so this is about checking visibility or not with the LookupMethodNode node then, and not fixed yet (currently LookupMethodNode always checks the visibility).